### PR TITLE
feat: add deprecated AEB token warning

### PR DESCRIPTION
**Pull request by SarjuHansaliya**
- if user select old aeb token in To input

old_token

- if user select aeb bridge token list

old_token_list

### CODE REVIEW

File: src/components/Warning/index.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <WarningWrapper>
+      <AutoColumn gap="4px">
+        <RowFixed>
+          <StyledWarningIcon />
+          <Text fontWeight={600} lineHeight={'145.23%'} ml={'10px'}>
+            Old AEB tokens Alert
+          </Text>
+        </RowFixed>
+        <Text fontWeight={500} lineHeight={'145.23%'} mt={'10px'}>
+          Please note these tokens were used by the old AEB bridge and have been deprecated. Using these tokens may
```
[the remark this answers is not in the document]
Comment (SarjuHansaliya) [reply src/components/Warning/index.tsx:43]: sure let me ask Justin in the group as message was given by him.

[the remark this answers is not in the document]
Comment (SarjuHansaliya) [reply src/components/Warning/index.tsx]: done

File: src/state/lists/hooks.ts [hunk 2]
```diff
@@ -134,3 +135,22 @@ export function useAllLists(): TokenList[] {
...
+export function useCheckAEBToken(): Boolean {
+  const listsByUrl = useSelector<AppState, AppState['lists']['byUrl']>(state => state.lists.byUrl)
+
+  const allAEBTokens = listsByUrl[AEB_TOKENLIST]?.current?.tokens || []
+
+  const selectedOutputToken = useSelector<AppState, AppState['swap']['OUTPUT']>(state => state.swap.OUTPUT)
+
+  const aebToken = allAEBTokens.find(token => token?.address === selectedOutputToken?.currencyId)
+  // ignore PNG and WAVAX token
+  if (aebToken?.symbol === 'PNG' || aebToken?.symbol === 'WAVAX') return false
```
[the remark this answers is not in the document]
Comment (SarjuHansaliya) [reply src/state/lists/hooks.ts]: done

File: src/state/lists/hooks.ts [hunk 3]
```diff
@@ -134,3 +135,22 @@ export function useAllLists(): TokenList[] {
...
   )
 }
+
+export function useIsSelectedAEBTokenList(): Boolean {
+  const selectedListUrl = useSelectedListUrl()
+  const isSelected = (selectedListUrl || []).includes(AEB_TOKENLIST)
+  return isSelected
+}
+
+export function useCheckAEBToken(): Boolean {
```
[the remark this answers is not in the document]
Comment (SarjuHansaliya) [reply src/state/lists/hooks.ts]: done

**Review by bmino — APPROVED** [review]
lgtm

**Pull request merged**